src/client/components/GatewayErrorSummary.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import { Meta } from '@storybook/react';
+import React from 'react';
+import GatewayErrorSummary from '@/client/components/GatewayErrorSummary';
+import { StructuredGatewayError } from '@/shared/model/Errors';
+
+export default {
+	title: 'Components/GatewayErrorSummary',
+	component: GatewayErrorSummary,
+	parameters: {
+		layout: 'padded',
+	},
+} as Meta;
+
+export const Default = () => (
+	<GatewayErrorSummary
+		gatewayError="Literally everything went wrong"
+		shortRequestId="123e4567"
+		context="Not a single thing went right"
+	></GatewayErrorSummary>
+);
+Default.storyName = 'default';
+
+export const WithoutContextOrRequestId = () => (
+	<GatewayErrorSummary gatewayError="something went wrong but we have no context about it."></GatewayErrorSummary>
+);
+
+const bauError: StructuredGatewayError = {
+	message: 'You entered a field wrong or something',
+	severity: 'BAU',
+};
+
+export const WithStructuredBAUError = () => (
+	<GatewayErrorSummary
+		gatewayError={bauError}
+		context={
+			<p>since this is a BAU error we do not display the request id here</p>
```

Comment:
These stories probably need to correct styling applied to them

```suggestion
			<p css={[errorContextSpacing, errorContextLastTypeSpacing]}>since this is a BAU error we do not display the request id here</p>
```